scripting/2.0/modules/ROOT/pages/scripting-module-upgrade-migrate.adoc

```diff
@@ -0,0 +1,109 @@
+= Upgrade Scripting Module to Version 2.0 - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Upgrade Scripting Module to version 2.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|1.1.7 |2.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Scripting Module no longer provides default scripting engines:
+** Use the *Configure...* helper button to install your desired engines.
+** Specify the desired engine in the `pom.xml` file of your application.
+* For any data stream provided in the payload or variables, a cursor object opens and binds, preventing the need for you to open new cursors. Any `openCursor()` provided to the Scripting Module closes after the script execution.
+
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must complete the following:
+
+* Create a backup of your files, data, and configurations in case you need to restore to them.
+
+== Upgrade Steps
+
+Follow these steps to perform the upgrade:
+
+. In Anypoint Studio, create a Mule project.
+. In *Mule Palette*, click *Search in Exchange*.
+. In *Add Dependencies to Project*, enter  in the search field.
+. In *Available modules*, select `Scripting Module` and click *Add*.
+. Click *Finish*.
+. Verify that the `mule-scripting-module` dependency version is `2.0.0` in the file in the Mule project.
+
+Studio upgrades the connector automatically.
```

Comment:
This may be confusing, because seems that Studio is upgrading the connector and their configs. I would be more explicit of "Studio bumps the version only"